### ISSUE TITLE
1.12.0 - Scoreboard

### ISSUE THREAD

**Issue by zachun** [urnotnick/impulse-issues#151]
**Describe the bug**
The bodygroups of the playermodel showed on the right side of the screen when clicking on a player do not match the ones that the said player actually has.

**To reproduce**

1. Get a second player on CP
2. Get them to set their rank and division
3. Go to the scoreboard and click on them

**Expected behaviour**
The bodygroups of the playermodel to match the ones of the actual player

**Screenshots**
![image](https://user-images.githubusercontent.com/72348423/219898763-1fba8137-df9d-4d26-9ed6-1bc46299d420.png)

**impulse version (press F1 in the bottom right):**
1.12.0

**Comment by johnnyroastbeef** [comment]
I dont think this is a bug I believe it is intended so they don't metagame the unit
